src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1318,11 +1330,22 @@ protected override void HandleScopesRequestAsync(IRequestResponder<ScopesArgumen
                         response.Scopes.Add(new Scope()
                         {
                             Name = AD7Resources.Locals_Scope_Name,
-                            VariablesReference = m_variableManager.Create(frame),
+                            VariablesReference = m_variableManager.Create(new VariablesRef() { StackFrame = frame, Scope = VariablesScope.Locals }),
+                            PresentationHint = Scope.PresentationHintValue.Locals,
                             Expensive = false
                         });
                     }
                 }
+
+                // registers should always be present
+                // and it's too expensive to read all values just to add the scope
```

Comment:
you could argue the Locals check should be avoided too

Reply:
@gregg-miskelly ?

Reply:
I wouldn't expect it to make too much of a difference because I would expect that if a client make a `scopes` request it is going to turn around and request children from each returned scope where `Expensive` is null or false. But I also don't know why this check was made -- it seems totally reasonable to always display locals and just have it be empty if there aren't any.